### ISSUE TITLE
Add ability to specify sorting mode for resource listing

### ISSUE THREAD

**Issue by Phoenix616** [SpigotMC/XenforoResourceManagerAPI#45]
The new resource listing endpoint should provide a way to sort/order the list in the same ways the website provides (as originally requested in #32 :eyes:)

**Comment by simpleauthority** [comment]
Yep my bad, will make a PR for that soon. Sorry. Thanks for making a new issue Phoenix

**Comment by simpleauthority** [comment]
Adding better sort params is next on my list. Sorry for the long wait.
